Lombiq.DataTables/Tests/Lombiq.DataTables.Tests/UnitTests/Services/ExportTests.cs

```diff
@@ -246,4 +241,50 @@ public static IEnumerable<object[]> Data()
             0,
         };
     }
+
+    // Sometimes a font is available however, it's corrupted or missing a table (for example, this can happen on
+    // GitHub-hosted runners). We can't check directly if a font is missing a table or corrupted, but we can try
+    // other fonts if this happens.
```

Comment:
```suggestion
    // Sometimes a font is available, however, it's corrupted or missing a table (for example, this can happen on
    // GitHub-hosted runners). We can't check directly if a font is missing a table or corrupted, but we can try
    // other fonts if this happens.
```